CONTRIBUTING.md

```diff
@@ -26,6 +26,17 @@ Code for Fastify's **v1.x** is in [branch 1.x](https://github.com/fastify/fastif
 
 Declaring formal releases remains the prerogative of the lead maintainers. Do not bump version numbers in pull requests.
 
+## Plugins
+
+The contributors to the Fastify's plugins must attend the same rules of the Fastify repository with few adjustements:
+
+1. A release can be published by any member.
+1. The plugin version must follow the SemVer specification.
+1. The plugin master branch must support the same Node.js versions of the master branch in the Fastify's repository.
```

Comment:
"The Node.js compatibility must match with the Fastify repository." This sounds much cleaner to me.

Reply:
I left the `master branch` since it was a suggestion from delvedor